app/views/scientists/show.html.erb

```diff
@@ -0,0 +1,14 @@
+<h1><%= @scientist.name %></h1>
+
+<p>Name: <%= @scientist.name %></p>
+<p>Specialty: <%= @scientist.specialty %></p>
+<p>University: <%= @scientist.university %></p>
+<p>Lab: <%= @scientist.lab.name %></p>
+<p>Experiments: </p>
+<li> 
+  <% @scientist.experiments.each do |experiment| %>
+    <%= "#{experiment.name}" %>
+    <%= button_to "Remove Experiment", "/scientists/#{@scientist.id}?experiment_id=#{experiment.id}", method: :delete %>
```

Comment:
Because your route is incorrect in your routes.rb file for the DELETE verb, it forces you to have to jerry-rig the experiment ID in the path somehow to get it to the controller. With a nested route, both IDs would fit nicely inside the path without having to use query params.